WalletWasabi.Tests/UnitTests/Transactions/SmartTransactionTests.cs

```diff
@@ -201,8 +201,8 @@ public void SmartTransactionLineDeserialization()
 			}
 
 			// Null and empty arguments.
-			string input = $"{txHash}:{txHex}:{height}:{blockHash}:{blockIndex}:{label}:{unixSeconds}:{isReplacement}";
-			Network network = null;
+			string? input = $"{txHash}:{txHex}:{height}:{blockHash}:{blockIndex}:{label}:{unixSeconds}:{isReplacement}";
+			Network? network = null;
 			Assert.Throws<ArgumentNullException>(() => SmartTransaction.FromLine(input, network));
```

Comment:
I would simply remove these lines for the reason Adam already said.

Reply:
I will do that later on, i got some work to do before that.